HAPNodeJSClient.js

```diff
@@ -462,18 +493,19 @@ HAPNodeJSClient.prototype.HAPevent = function(ipAddress, port, body, callback) {
  * HAPNodeJSClient.prototype.HAPresourceByDeviceID - Send a characteristic PUT Message to a particular homebridge instance using resource interface, ie camera
  *
  * @param  {type} DeviceID  DeviceID of homebridge instance
+ * @param  {type} pin       Pin of homebridge instance
  * @param  {type} body      An array of HomeKit characteristic updates, [{ \"aid\": 2, \"iid\": 9, \"value\": 0}]
  * @param  {type} callback  Callback to execute upon completion of characteristic setting, function(err, response)
  */
 
-HAPNodeJSClient.prototype.HAPresourceByDeviceID = function(deviceID, body, callback) {
+HAPNodeJSClient.prototype.HAPresourceByDeviceID = function(deviceID, pin, body, callback) {
```

Comment:
This is a breaking change to the external API and can not be merged as it will break all other packages using this package